src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -472,6 +477,7 @@ private MessageConsumer newInitialStream(
             final long affinity = begin.affinity();
 
             newStream = new AmqpServer(sender, routeId, initialId, affinity)::onNetwork;
+            this.connectionState = START;
```

Comment:
The state should be stored on the `server` since we can be decoding more than one `AmqpServer` in parallel.